config/hubs/meom-ige.cluster.yaml

```diff
@@ -75,7 +75,7 @@ hubs:
                 description: "~32 CPU, ~128G RAM"
```

Comment:
```suggestion
                description: "~32 CPU, ~100G RAM"
```

Reply:
Well, if we modify the description, it should be ~110G.

Reply:
I think the description should be what the node has capacity for, not what we guarantee.

The point of the guarantee is set to make sure we don't end up with multiple user pods on that node as I think of it. So if its ~128 or ~110 I don't know, but i recall 3.5G per CPU, so 112, but with a bit of memory less for system things, so ~110 seems about right for a n1 node.

Reply:
```suggestion
                description: "~32 CPU, ~110G RAM"
```